### PR TITLE
refac: add mapper classes in CalculationResult

### PR DESCRIPTION
**Pull request by JesperGomes**
## References

* #1023

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.UnitTests/Infrastructure/CalculationResultClient/Mappers/AggregationLevelMapperTest.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+public class AggregationLevelMapperTests
+{
+    [Theory]
+    [InlineData(TimeSeriesType.Production)]
+    [InlineData(TimeSeriesType.FlexConsumption)]
+    [InlineData(TimeSeriesType.NonProfiledConsumption)]
+    public void ToDeltaTable_WhenNoActorsSpecified_ReturnsExpectedAggLevel(TimeSeriesType type)
+    {
+        // Act
+        const string expected = "total_ga";
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.UnitTests/Infrastructure/CalculationResultClient/Mappers/AggregationLevelMapperTest.cs:33]: General observation: All these expected values can easily change in delta/pyspark and thus break the .NET stack.
I think we should add contract tests too (in another PR?).

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/Mappers/TimeSeriesTypeMapper.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResultClient;
+
+namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResultClient.Mappers;
+
+public static class TimeSeriesTypeMapper
+{
+    public static TimeSeriesType FromDeltaTable(string timeSeriesType) =>
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/Mappers/TimeSeriesTypeMapper.cs:21]: General naming observation: I was confused about the naming at first as "TimeSeriesTypeMapper.FromDeltaTable(...)" tells me that by calling this method I should provide a delta table and will get time series type in return - and vice versa regarding "ToDeltaTable".

I like the "To" and "From" prefixing. But can we come up with another name for the remainder? "FromDeltaTableValue"? "FromStorageValue"? ...?

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.UnitTests/Infrastructure/CalculationResultClient/Mappers/AggregationLevelMapperTest.cs (hunk 1 above)
Comment (JesperGomes) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.UnitTests/Infrastructure/CalculationResultClient/Mappers/AggregationLevelMapperTest.cs:33]: Yes - completely agree. I was considering if I should make it as a separate PR.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/Mappers/TimeSeriesTypeMapper.cs (hunk 2 above)
Comment (JesperGomes) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/Mappers/TimeSeriesTypeMapper.cs]: Now renamed to FromDeltaTableValue

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**